brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/CsvTrainer.scala

```diff
@@ -0,0 +1,74 @@
+package com.stripe.brushfire
+package scalding
+
+import com.stripe.brushfire.features._
+
+import com.twitter.scalding._
+import com.twitter.algebird.AveragedValue
+
+class CsvTrainerJob(args: Args) extends TrainerJob(args) {
+  import JsonInjections._
+
+  implicit val stopper = FrequencyStopper[String](10, 3)
+  implicit val errorOrdering = Ordering[Double].on[AveragedValue] { av: AveragedValue => av.value } // Allows us to compare BrierScoreErrors.
+
+  val error = BrierScoreError[String, Long]
+
+  def splitRow(row: String): CsvRow = row.split(",").map(_.trim)
+
+  val header: CsvRow = splitRow(args("features"))
```

Comment:
this seems to be `columns` rather than `features` since it also includes the label feature